followed-accounts.js

```diff
@@ -43,10 +43,9 @@ class FollowedActorsList extends HTMLElement {
     const followedActors = await db.getFollowedActors()
     this.innerHTML = ''
     followedActors.forEach((actor) => {
-      const actorElement = document.createElement('div')
-      const formattedDate = this.formatDate(actor.followedAt)
-
-      actorElement.innerText = `- Followed URL: ${actor.url} - Followed At: ${formattedDate}`
+      const actorElement = document.createElement('actor-mini-profile')
```

Comment:
![Screenshot 2024-03-30 at 2 12 35 AM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/495dd823-1755-490b-a35b-a8499f851947)

Reply:
Yeah this looks great!